phoenix-core/src/it/java/org/apache/phoenix/end2end/ViewUtilIT.java

```diff
@@ -443,4 +445,108 @@ public void testOrphanViewDetection() throws Exception {
+        tenantProps.setProperty(TENANT_ID_ATTRIB, tenantId);
+        final String tenantViewName1 = schemaPrefix + generateUniqueName();
+        final String tenantViewName2 = schemaPrefix + generateUniqueName();
+        final String tenantViewIndex11 = generateUniqueName() + "_INDEX11";
+        final String tenantViewIndex21 = generateUniqueName() + "_INDEX21";
+        final String tenantViewIndex22 = generateUniqueName() + "_INDEX22";
+
+        final String createTableDDL = "CREATE TABLE " + tableName +
+                "(TENANT_ID CHAR(10) NOT NULL, ID CHAR(10) NOT NULL, NUM BIGINT " +
+                "CONSTRAINT PK PRIMARY KEY (TENANT_ID, ID)) MULTI_TENANT=true";
+        final String globalViewDDL = "CREATE VIEW %s (PK1 BIGINT, PK2 BIGINT) " +
+                "AS SELECT * FROM " + tableName + " WHERE NUM > -1";
+        final String viewIndexDDL = "CREATE INDEX %s ON %s (NUM DESC) INCLUDE (ID)";
+        final String tenantViewDDL = "CREATE VIEW %s AS SELECT * FROM %s";
+
+        try (Connection conn = DriverManager.getConnection(getUrl())) {
+
+            // Create data table, global views and global view indexes
+            conn.createStatement().execute(createTableDDL);
+            conn.createStatement().execute(String.format(globalViewDDL, globalViewName1));
+            conn.createStatement().execute(String.format(globalViewDDL, globalViewName2));
+            conn.createStatement().execute(String.format(viewIndexDDL, globalViewIndex11, globalViewName1));
+            conn.createStatement().execute(String.format(viewIndexDDL, globalViewIndex12, globalViewName1));
+            conn.createStatement().execute(String.format(viewIndexDDL, globalViewIndex21, globalViewName2));
+            conn.createStatement().execute(String.format(viewIndexDDL, globalViewIndex22, globalViewName2));
+            conn.createStatement().execute(String.format(viewIndexDDL, globalViewIndex23, globalViewName2));
+
+
+            // Create tenant views and tenant view indexes
+            try (Connection tenantConn = DriverManager.getConnection(getUrl(), tenantProps)) {
+                tenantConn.createStatement().execute(
+                        String.format(tenantViewDDL, tenantViewName1, tableName));
+                tenantConn.createStatement().execute(
+                        String.format(tenantViewDDL, tenantViewName2, tableName));
+                tenantConn.createStatement().execute(String.format(viewIndexDDL, tenantViewIndex11, tenantViewName1));
+                tenantConn.createStatement().execute(String.format(viewIndexDDL, tenantViewIndex21, tenantViewName2));
+                tenantConn.createStatement().execute(String.format(viewIndexDDL, tenantViewIndex22, tenantViewName2));
+            }
+
+            // Get view indexes ids only for global view indexes (excluding tenant view indexes)
+            List<String> list = ViewUtil.getViewIndexIds(conn.unwrap(PhoenixConnection.class), "_IDX_" + tableName, false);
```

Comment:
use `MetatadataUtil.getViewIndexPhysicalName` instead of using hardcoded constants _IDX